tmt/steps/provision/mrack.py

```diff
@@ -528,25 +527,24 @@ def options(cls, how: Optional[str] = None) -> List[tmt.options.ClickOptionDecor
         """ Prepare command line options for Beaker """
         return [
             click.option(
-                '--arch', metavar='ARCH',
-                help='Architecture to provision.'
-                ),
+                '--arch',
+                metavar='ARCH',
+                help='Architecture to provision.'),
             click.option(
-                '--image', metavar='COMPOSE',
-                help='Image (distro or "compose" in Beaker terminology) '
-                     'to provision.'
-                ),
+                '--image',
+                metavar='COMPOSE',
+                help='Image (distro or "compose" in Beaker terminology) to provision.'),
             click.option(
-                '--provision-timeout', metavar='SECONDS',
-                help=f'How long to wait for provisioning to complete, '
-                     f'{DEFAULT_PROVISION_TIMEOUT} seconds by default.'
-                ),
+                '--provision-timeout',
+                metavar='SECONDS',
+                help=f'How long to wait for provisioning to complete, {DEFAULT_PROVISION_TIMEOUT} '
+                     f'seconds by default.'),
             click.option(
-                '--provision-tick', metavar='SECONDS',
-                help=f'How often check Beaker for provisioning status, '
-                     f'{DEFAULT_PROVISION_TICK} seconds by default.',
-                ),
-            ] + super().options(how)
+                '--provision-tick',
+                metavar='SECONDS',
+                help=f'How often check Beaker for provisioning status, {DEFAULT_PROVISION_TICK} '
+                     f'seconds by default.'),
```

Comment:
```suggestion
                     'seconds by default.'),
```

Reply:
This took care of itself, when I was reverting the formatting changes :) Thanks though!